packages/web-components/src/utils/utils.ts

```diff
@@ -58,14 +58,15 @@ export const renderHiddenInput = (
     name: string,
     value: string | undefined | null,
     disabled: boolean,
-    checked?: boolean
+    checked?: boolean,
+    secondInput?: boolean
 ) => {
     let input = container.querySelector(
         'input.aux-input'
```

Comment:
did you mean `input.rux-input`?

Reply:
No, it's `aux-input`. It's a hidden, native input element that just facilitates form submissions since shadow-dom has probs with that.